pkg/mdformatter/linktransformer/validator.go

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) Bartłomiej Płotka @bwplotka
+// Licensed under the Apache License 2.0.
+
+package linktransformer
+
+import (
+	"encoding/json"
+	"fmt"
+	"math"
+	"net/http"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+const (
+	gitHubAPIURL = "https://api.github.com/repos/%v/%v?sort=created&direction=desc&per_page=1"
+)
+
+type GitHubResponse struct {
+	Number int `json:"number"`
+}
+
+// Match link with any one of provided validators.
+func CheckValidators(dest string, v Config) bool {
```

Comment:
```suggestion
func (v Config).GetValidatorForURL(url string) SomeValidator interface {
```

Let's have some interface for each validator so it's cleaner, WDYT?

Reply:
Sure!